CONTRIBUTING.md

```diff
@@ -0,0 +1,131 @@
+  * Check the [coding standards](#coding-standards) section to ensure that you
+    have followed the coding conventions of GatorGrouper.
+
+## Contributing features
+
+If you have implemented a new feature for GatorGrouper:
+  * Check the Issue Tracker for any issues relevant to your implemented feature.
+  * [Open a new pull request](https://github.com/GatorEducator/gatorgrouper/compare).
+    Write a **clear description** of the feature and its usage. In your pull
+    request, be sure to have updated any relevant documentation to reflect the
+    usage of the tool with your implemented features.
+  * Be sure that you have followed the [coding standards](#coding-standards)
+    outlined below for any code contributions you have made.
+
+## Coding standards
+
+By contributing source code to the GatorGrouper project, you are agreeing that
+any code you have written follows the coding standards outlined below. This
+means adhering to the style, testing, and complexity standards as follows.
+
+### Code style
+
+All GatorGrouper code should be written in Python. Code written for GatorGrouper
+should,
+  * Be written in a modular fashion, making use of functions and modules.
+  * Be clearly documented with both docstring comments describing functions, as
+    well as single line comments describing particularly complex lines of code.
+  * Pass `flake8` checks without errors. You can learn more about using `flake8`
+    [here](http://flake8.pycqa.org/en/latest/).
+  * Score 10.0/10.0 when using the `pylint` tool. You can learn more about using
+    `pylint` [here](https://www.pylint.org/).
+
+It is strongly encouraged that you use the Python code formatter,
+[black](https://github.com/ambv/black). This will help with meeting the above
+requirements for both `flake8` and `pylint`.
+
+### Testing
+
+Test cases for any implemented code **must** be provided. GatorGrouper uses the
+`pytest` automated testing suite. `pytest` test cases should,
+  * Cover the entirety of the code that is being executed. This means that there
+    should be test cases for each and every function or method inside the
+    program.
+  * Be able to handle mutation testing to ensure that the test cases are
+    accurate.
+  * Have clear docstrings and doctests that allow developers to know what is
+    being tested.
+
+### Minimizing complexity
+
+The definition of complexity we use is taken from “A Philosophy of Software
+Design” by Chairman of Electric Cloud and Professor of Computer Science at
+Stanford, John Ousterhout. John defines complexity as anything “related to the
+structure of a software system that makes it hard to understand and modify the
+system.” To this end, the standards here aim to minimize complexity by
+maximizing the clarity and ease of modification of GatorGrouper. All
+contributions to GatorGrouper should have the following qualities:
+
+1. Changeability -
+*Software will be easily changeable to allow for future scalability.*
```

Comment:
Should this use *`shall`*?

Reply:
That, it should. Good catch! Fixed in a commit.